File: Practice/30 Days of Code/Day 14 Scope/Day 14 Scope.php
<?php
    class Difference{
    private $elements=array();
    public $maximumDifference;

// Write your code here
function __construct($arr) {
        $this->elements = $arr;
    }
    public function ComputeDifference() {
        $max = 1;
        $min = 100;
        foreach($this->elements as $num) {
            if( $num > $max ) $max = $num;
            if( $num < $min ) $min = $num;
        }
        $this->maximumDifference = $max - $min;
    }
} //End of Difference class


$N=intval(fgets(STDIN));
$a =array_map('intval', explode(' ', fgets(STDIN)));
$d=new Difference($a);
$d->ComputeDifference();
print ($d->maximumDifference);
?>

File: Practice/30 Days of Code/Day 14 Scope/Day 14 Scope.py
class Difference:
    def __init__(self, a):
        self.__elements = a

    # Add your code here
    def computeDifference(self):
        mx = 1
        mn = 100
        for num in self.__elements:
            if( num > mx ): mx = num
            if( num < mn ): mn = num
        self.maximumDifference = mx - mn
# End of Difference class